dax-prompts/dax-prompts-impl/src/main/java/com/duckduckgo/daxprompts/impl/RealDaxPrompts.kt

```diff
@@ -16,27 +16,64 @@
 
 package com.duckduckgo.daxprompts.impl
 
+import com.duckduckgo.browser.api.UserBrowserProperties
 import com.duckduckgo.common.utils.DispatcherProvider
 import com.duckduckgo.daxprompts.api.DaxPrompts
+import com.duckduckgo.daxprompts.api.DaxPrompts.ActionType
+import com.duckduckgo.daxprompts.impl.ReactivateUsersToggles.Cohorts.CONTROL
+import com.duckduckgo.daxprompts.impl.ReactivateUsersToggles.Cohorts.VARIANT_1
+import com.duckduckgo.daxprompts.impl.ReactivateUsersToggles.Cohorts.VARIANT_2
 import com.duckduckgo.daxprompts.impl.repository.DaxPromptsRepository
 import com.duckduckgo.di.scopes.AppScope
 import com.squareup.anvil.annotations.ContributesBinding
 import dagger.SingleInstanceIn
+import java.util.Date
 import javax.inject.Inject
 import kotlinx.coroutines.withContext
 
+private const val EXISTING_USER_DAY_COUNT_THRESHOLD = 28
+private const val EXISTING_USER_DAYS_INACTIVE_MILLIS = 7 * 24 * 60 * 60 * 1000 // 7 days
+
 @SingleInstanceIn(AppScope::class)
 @ContributesBinding(AppScope::class, boundType = DaxPrompts::class)
 class RealDaxPrompts @Inject constructor(
     private val daxPromptsRepository: DaxPromptsRepository,
+    private val reactivateUsersToggles: ReactivateUsersToggles,
+    private val userBrowserProperties: UserBrowserProperties,
     private val dispatchers: DispatcherProvider,
 ) : DaxPrompts {
 
-    override suspend fun shouldShowDuckPlayerPrompt(): Boolean = withContext(dispatchers.io()) {
+    override suspend fun evaluate(): ActionType {
+        return withContext(dispatchers.io()) {
+            if (!reactivateUsersToggles.reactivateUsersExperimentMay25().isEnrolled() && !isEligible()) {
+                return@withContext ActionType.NONE
+            }
```

Comment:
This is always gonna go through here. At this point none of the users will be enrolled so this is always false. I think you just want to return none if the user is not eligible.